Update TwitterBot.py

jdygert opened the pull request:
Nice bot! Here's some suggestions for refactoring this code.

Commit:
Update TwitterBot.py

sharadbhat reviewed (changes requested):
Could you make a small change so that it uploads once every four hours? 
Other than that, it's 10x better than my original code.

Commit:
Change posting to every 4 hours

I'm not sure I like the way I did it, but this will make it post at 0, 4, 8, 12, 16, and 20 hours of the day.

sharadbhat reviewed (approved):
Yeah. This is good. Thanks
